persistent/Database/Persist/Quasi/Internal/ModelParser.hs

```diff
@@ -211,19 +211,18 @@ tryOrWarn msg p l r = do
             else parseError err
 
 -- | Attempts to parse with a provided parser. If it fails with an error matching
--- the provided predicate, it registers a delayed error with the provided message and falls
+-- the provided predicate, it registers a delayed error and falls
```

Comment:
This comment was inaccurate; the `msg` argument was actually unused. I'm not sure that's what we actually want, but fixing it seemed outside the scope of this PR.